lib/plot/falco_plot_progress_dst.m

```diff
@@ -8,7 +8,7 @@
 
 tb = mp.tb;
 
-if(Itr==10 || Itr==40)
+if(Itr==10 || Itr==40 && ~mp.flagSim)
```

Comment:
Please add another set of parentheses so the logic is clearer. I can't tell right now if it is doing `(A or B) and C`, or `A or (B and C)`.